### PR TITLE
Add Python xmlsec wheel (py3-xmlsec)

### PR DESCRIPTION
**Pull request by johnfosborneiii**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

File: py3-xmlsec [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    with:
+      expected-commit: b483b644b6033e539f303509f5ee85bebf178a73
+      repository: https://github.com/xmlsec/python-xmlsec
+      tag: ${{package.version}}
+
+  - runs: |
+      set -ex
+      # Patch for open issue - https://github.com/xmlsec/python-xmlsec/issues/252
+      # https://github.com/xmlsec/python-xmlsec/pull/258
+      git pull origin pull/258/head
```
[the remark this answers is not in the document]
Comment (dlorenc) [reply py3-xmlsec:50]: Agree with @stormqueen1990 - just to note - you can access the .patch file using something like: `https://patch-diff.githubusercontent.com/raw/xmlsec/python-xmlsec/pull/258.patch` and then check in the patch

[the remark this answers is not in the document]
Comment (johnfosborneiii) [reply py3-xmlsec]: Done!

**Review by stormqueen1990 — COMMENTED** [review]
LGTM!

File: py3-xmlsec.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      tag: ${{package.version}}
+
+  - uses: patch
+    with:
+      # Patch for open issue - https://github.com/xmlsec/python-xmlsec/issues/252
+      # https://github.com/xmlsec/python-xmlsec/pull/258
+      patches: pull_258.patch
+
+  - name: Python Install
+    runs: pip3 install . --root=${{targets.destdir}}
```
Comment (kaniini) [inline py3-xmlsec.yaml:53]: Why not gpep517 here?  Using `pip` can be dangerous as it will vendor modules.

Comment (johnfosborneiii) [reply py3-xmlsec.yaml]: Done. Switching to gpep517 also found a missing runtime dependency.

**Review by kaniini — APPROVED** [review]

**Pull request merged**